edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/pvc/KubernetesPvcMapper.cs

```diff
@@ -32,7 +32,7 @@ public class KubernetesPvcMapper : IKubernetesPvcMapper
 
         bool ShouldCreatePvc(Mount mount)
```

Comment:
Should the type be `Mount?` instead of `Mount` ?

Reply:
Since we are not currently using [Nullable Reference Types](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-reference-types), it is not necessary, since objects can be null.